README.adoc

```diff
@@ -24,23 +24,28 @@ Learn how to use JAX-RS, CDI, and JSON-P to build a MicroProfile application.
 
 == What you'll learn
 
-MicroProfile is an initiative to optimize Enterprise Java for a microservices architecture.
-You will write REST web services with JAX-RS and JSON-P and also use CDI to inject dependencies
-and help manage the lifecycle of your application.
-
-You will create an application that provides an `inventory` service, which allows you to retrieve
-the information for a particular host or a list of all previously registered hosts.
-If the requested host is not in the inventory, the `inventory` service attempts to retrieve
-that host's information by calling its `system` service.
-
-For example, you will be able to access the following URL to retrieve a list of all hosts in the inventory:
-
+In this guide, you will learn how to write RESTful web services using JAX-RS and JSON-P while utilizing
```

Comment:
What the rationale to remove the line that explains MicroProfile?

Reply:
it was weird for the title to be "What you'll learn" and then immediately go into explaining what microprofile is. I wanted to move that section somewhere else, or lower at least, I guess I just forgot. Anyway, we also always started our guides with "In this guide, you will learn..."

Reply:
fixed

Reply:
Can you remove 'In this guide' (it's not necessary; just say 'You will learn...').

I agree that it should be re-reviewed/tested if it's significantly changed.

I was discussing this guide with @NottyCode  and @gcharters earlier in the week after reviewing part of it myself and we were concerned about the length of it and that the text needs a tighter focus to be about integrating JAX-RS with CDI but without some of the duplication of the JAX-RS and Maven guides. I need to look at it with @gcharters though to know just what should be updated (as he knows the area better and what he and Alasdair intended).